jsk_pcl_ros/scripts/voxel_urdf_client.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python
+import rospy
+import argparse
+import os
+
+PKG='jsk_pcl_ros'
+
+from visualization_msgs.msg import Marker
+from jsk_pcl_ros.srv import VoxelModelGenerate
+
+def markerCallback(msg):
+    global marker_msg
+    marker_msg = msg
+
+def main():
+    global marker_msg
+    rospy.init_node("voxel_urdf_client")
+    # parse argument
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--gazebo', default=False)
+    parser.add_argument('--model', default=False)
+    parser.add_argument('--loop', default=True)
+    args = parser.parse_args(rospy.myargv()[1:])
+
+    marker_sub = rospy.Subscriber("input", Marker, markerCallback)
+    generate_srv = rospy.ServiceProxy("/generate_voxel_urdf", VoxelModelGenerate)
+
+    while args.loop:
```

Comment:
moved to callback.
When `--loop` is false, I'd like to finish this script after generating a model once.